test/e2e/functional/experiment-with-service.yaml

```diff
@@ -26,5 +26,6 @@ spec:
                   memory: 16Mi
                   cpu: 1m
               ports:
-                - protocol: TCP
+                - name: testport
```

Comment:
does the test fail when name is not present? can we rather have two tests, one with single, unnamed port, another one with two named ports?

Reply:
Yes, test fails when name isn't present. I'll try that out

Reply:
okay, so let's make sure the test doesn't fails without name first